features/FxA-54-push-on-status-change/README.md

```diff
@@ -30,6 +30,9 @@ As a concrete metric
 we can measure the number of calls to `/recovery_email/status`
 and check that it decreases as this feature rolls out.
 
+We will also add additional metrics to signal when Desktop verifies 
+via Push.
```

Comment:
Excellent note, I will add that in and merge. Please follow up if something is not correct.